src/plugins/discover/public/application/components/default_discover_table/helper.tsx

```diff
@@ -55,11 +56,16 @@ export interface ColumnProps {
  * prepended, not moveable and removeable
  * @param timeFieldName
  */
-export function getTimeColumn(timeFieldName: string): ColumnProps {
+export function getTimeColumn(
+  timeFieldName: string,
+  osdFieldOverrides: {
+    [key: string]: any;
+  }
+): ColumnProps {
   return {
     name: timeFieldName,
     displayName: 'Time',
-    isSortable: true,
+    isSortable: osdFieldOverrides.sortable == null ? true : osdFieldOverrides.sortable,
```

Comment:
nitpick: 
```suggestion
    isSortable: osdFieldOverrides.sortable ?? true
```

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing